src/app_service/service/chat/service.nim

```diff
@@ -204,24 +203,14 @@ QtObject:
         chats.add(chat)
     result = (chats, messages)
 
-  proc processMessageUpdateAfterSend*(self: Service, response: RpcResponse[JsonNode]): (seq[ChatDto], seq[MessageDto])  =
+  proc processMessageUpdateAfterSend*(self: Service, response: RpcResponse[JsonNode]): (seq[ChatDto], seq[MessageDto]) =
     result = self.parseChatResponse(response)
     var (chats, messages) = result
     if chats.len == 0 or messages.len == 0:
       error "no chats or messages in the parsed response"
       return
 
-    # The reason why we are sending all the messages with responseTo filled in is because
```

Comment:
hey @MishkaRogachev this as far as I remember was done cause the reply wasnt coming filled correctly from status_go and we had to manually update it in some cases, i do not remember the exactly use case that this didnt work in though

Reply:
Thanks) I've tested with replies and it seems to be working well. I will wait for @saledjenic probably he remember some use case

Reply:
@MishkaRogachev I remember that I added that in the commit `58f10babba868fa2d2e99a779f8358b8d0ed3eac` for the old status-lib repo, which is not in use any more. And it was almost a year ago, how that ended up in the refactored code I am not sure, as I see it was added in the commit `a9c968b984d15f2c74e42689007d0fc88fb1086a `. All in all that is most outdated code, so you are free to remove it and if that causes new issue you can log it and we can fix it according to the code we're currently maintaining.